### PR TITLE
Register Subdomain: raz

### PR DESCRIPTION
**Pull request by Raz-js**
Added the domain: raz

### CODE REVIEW

File: domains/raz.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "Raz-js",
+        "email": "21razebr@wbhs.org.za"
+      },
+      "record": {
+        "CNAME": "raz-js.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/raz.json:8]: Where is your website located?

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please tell us where your repo is or temporarily remove the CNAME file

**Review by wdhdev — CHANGES_REQUESTED** [review]
I can't reach your website

**Comment by WilliamDavidHarrison** [comment]
Unreachable website + no response

**Pull request closed without merging**